client/src/i18n/fr/report.json

```diff
@@ -136,7 +136,17 @@
       "TITLE" : "Rapport de changement d'inventaire",
       "DESCRIPTION": "Ce rapport montre les changements survenus sur les articles en stock au cours d'une période"
     },
-
+    "LOST_STOCK_REPORT": {
+      "TITLE": "Rapport sur les stocks perdus",
+      "DEPOT_ROLE": "List stock transfer losses for this depot as",
+      "DESCRIPTION": "Ce rapport répertorie les stocks perdus lors du transfert de stock d'un dépôt à un autre.",
+      "DESTINATION": "Destination",
+      "DESTINATION_DEPOT" : "Dépôt de destination",
+      "EITHER": "Soit",
```

Comment:
![image](https://user-images.githubusercontent.com/5445251/135258441-641c0772-a4e9-4b38-a3d9-61e3e58284d9.png)

If the context of either here is any, you can say: `L'un ou l'autre` since there are just two cases

Reply:
Fixed. Thanks!